frontend/src/DivWithProps.tsx

```diff
@@ -0,0 +1,10 @@
+import * as React from 'react';
+
+type Props = {
+  children: React.ReactNode;
+};
+const DivWithProps = (extraProps: {}) => (props: Props) => (
```

Comment:
huh. what is the purpose of this?

Reply:
it's a component maker for the times when you just need a div with some CSS on it, but don't want to give it a name/file